src/WikipediaBox.vala

```diff
@@ -42,12 +43,35 @@ namespace Hypatia {
 
             link_button = new Gtk.LinkButton.with_label("https://wikipedia.org", "Read more on Wikipedia");
             link_button.halign = Gtk.Align.CENTER;
+            
+            wike_button = new Gtk.Button.with_label("Open via Wike App") {
+            	css_classes = {"suggested-action","pill"},
+            	halign = Gtk.Align.CENTER
+            };
+            
+            wike_button.clicked.connect(()=>{
+            	string ls_stdout;
+                string ls_stderr;
+                int ls_status;
+
+                try {
+                    Process.spawn_command_line_sync("wike -u " + link_button.get_uri(),
```

Comment:
Rather than trying to spawn this from the command line, you should use [GLib.AppInfo.Launch](https://valadoc.org/gio-2.0/GLib.AppInfo.launch.html).